portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>Roland's - Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
+
+<nav id="navbar">
+	<div id="nav-left-side">
+		<a href="/index.html" class="nav-button navbar-item">NR</a>
```

Comment:
Did you want 2 links to the homepage?

Reply:
I will resolve it and have one single link to the homepage on the name